pkg/pod/entrypoint_test.go

```diff
@@ -150,11 +164,11 @@ const nopImage = "nop-image"
 // image.
 func TestStopSidecars(t *testing.T) {
 	stepContainer := corev1.Container{
-		Name:  StepPrefix + "my-step",
+		Name:  stepPrefix + "my-step",
 		Image: "foo",
 	}
 	sidecarContainer := corev1.Container{
-		Name:    SidecarPrefix + "-my-sidecar",
+		Name:    sidecarPrefix + "-my-sidecar",
```

Comment:
the most minor of nits: this will result in a double hyphen b/c `sidecarPrefix` includes one.